source/_integrations/template.markdown

```
@@ -48,20 +49,20 @@ related:
 
 The `template` integration allows creating entities which derive their values from other data. This is done by specifying [templates](/docs/configuration/templating/) for properties of an entity, like the name or the state.
 
-Alarm control panels, binary (on/off) sensors, buttons, images, numbers, selects, sensors, switches and weathers are covered on this page. They can be configured using [UI](#configuration) or [YAML](#yaml-configuration) file.
+Alarm control panels, binary sensors, buttons, covers, fans, images, lights, locks, numbers, selects, sensors, switches, and weathers are covered on this page. They can be configured using [UI](#configuration) or [YAML](#yaml-configuration) file.
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)